manifests/opsmanager/install.pp

```diff
@@ -0,0 +1,42 @@
+# PRIVATE CLASS: do not call directly
+class mongodb::opsmanager::install {
+  $package_ensure = $mongodb::opsmanager::package_ensure
```

Comment:
can you add `assert_private()` from stdlib as first statement after the class header?

Reply:
If I assert_private() do I then need to remove the ops_manager_install_spec.rb file? I am likely mistaken, but won't it prevent the tests from being able to run?

Reply:
The tests need to be moved to a file that tests the `mongodb::opsmanager` class.